spec.html

```diff
@@ -9543,10 +9542,10 @@
         &lt;ZWJ&gt;
 
       UnicodeIDStart ::
-        &gt; any Unicode code point with the Unicode property &ldquo;ID_Start&rdquo;
+        &gt; any Unicode code point with the Unicode property “ID_Start”
```

Comment:
Agreed, I didn't know this was a known issue in ecmarkup. Looks like there's even an open PR to fix this bug: https://github.com/bterlson/ecmarkup/pull/117